Exception when trying to start operate

I get the following exception when I try to start operate:

```
zeebe_1          | Caused by: java.lang.ClassNotFoundException: io.zeebe.exporter.ElasticsearchExporter
zeebe_1          |      at jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source) ~[?:?]
zeebe_1          |      at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source) ~[?:?]
zeebe_1          |      at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
zeebe_1          |      at io.camunda.zeebe.broker.exporter.repo.ExporterRepository.load(ExporterRepository.java:78) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.buildExporterRepository(Broker.java:435) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.lambda$partitionsStep$21(Broker.java:408) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.lambda$startStepByStep$2(StartProcess.java:63) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.startStepByStep(StartProcess.java:61) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.start(StartProcess.java:46) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.partitionsStep(Broker.java:423) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.lambda$initStart$10(Broker.java:234) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.lambda$startStepByStep$2(StartProcess.java:63) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.startStepByStep(StartProcess.java:61) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.takeDuration(StartProcess.java:92) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.bootstrap.StartProcess.start(StartProcess.java:46) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.internalStart(Broker.java:180) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.util.LogUtil.doWithMDC(LogUtil.java:21) ~[zeebe-util-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.Broker.start(Broker.java:160) ~[zeebe-broker-1.0.0.jar:1.0.0]
zeebe_1          |      at io.camunda.zeebe.broker.StandaloneBroker.run(StandaloneBroker.java:60) ~[camunda-cloud-zeebe-1.0.0.jar:1.0.0]
zeebe_1          |      at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:819) ~[spring-boot-2.4.5.jar:2.4.5]
zeebe_1          |      ... 5 more
zeebe_1          | 2021-05-18 06:28:30.783 [] [main] INFO
zeebe_1          |       org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor - Shutting down ExecutorService 'applicationTaskExecutor'
operate_1        | 2021-05-18 06:28:31.691  WARN 7 --- [           main] i.c.o.u.RetryOperation                   : method [HEAD], host [http://elasticsearch:9200], URI [/_index_template/operate_template?master_timeout=30s], status line [HTTP/1.1 405 Method Not Allowed]
operate_zeebe_1 exited with code 1
```

I use the latest master.

--- a/cluster/zeebe.cfg.toml
+++ b/cluster/zeebe.cfg.toml
@@ -248,7 +248,7 @@
 #   two exporters that point to the same JAR, with the same class or a different one, and use args
 #   to parametrize its instantiation.
 # className:
-#   entry point of the exporter, a class which *must* extend the io.zeebe.exporter.Exporter
+#   entry point of the exporter, a class which *must* extend the io.camunda.zeebe.exporter.Exporter
 #   interface.
 #
 # A nested table as [exporters.args] will allow you to inject arbitrary arguments into your
@@ -279,7 +279,7 @@
 #
 #[[exporters]]
 #id = "elasticsearch"
-#className = "io.zeebe.exporter.ElasticsearchExporter"
+#className = "io.camunda.zeebe.exporter.ElasticsearchExporter"
 #
 #  [exporters.args]
 #  url = "http://localhost:9200"

--- a/operate-simple-monitor/application.yaml
+++ b/operate-simple-monitor/application.yaml
@@ -88,7 +88,7 @@ zeebe:
     #
     exporters:
       elasticsearch:
-        className: io.zeebe.exporter.ElasticsearchExporter
+        className: io.camunda.zeebe.exporter.ElasticsearchExporter
 
         args:
           url: http://elasticsearch:9200

--- a/operate-simple-monitor/docker-compose.yml
+++ b/operate-simple-monitor/docker-compose.yml
@@ -49,7 +49,7 @@ services:
       - zeebe_network
   elasticsearch:
     container_name: elasticsearch
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.7.1
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.2
     ports:
       - "9200:9200"
     environment:

--- a/operate/application.yaml
+++ b/operate/application.yaml
@@ -88,7 +88,7 @@ zeebe:
     #
     exporters:
       elasticsearch:
-        className: io.zeebe.exporter.ElasticsearchExporter
+        className: io.camunda.zeebe.exporter.ElasticsearchExporter
 
         args:
           url: http://elasticsearch:9200

--- a/operate/docker-compose.yml
+++ b/operate/docker-compose.yml
@@ -34,7 +34,7 @@ services:
     networks:
       - zeebe_network
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.7.1
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.2
     ports:
       - "9200:9200"
     environment:

